www/js/app.js

```diff
@@ -64,6 +61,87 @@ angular.module('emission', ['ionic',
             Logger.log("error "+JSON.stringify(err)+" reading or setting defaults, giving up");
         });
     });
+
+    Logger.log("Setting collection settings to default");
+    var androidDefaultCollectionConfig = {
```

Comment:
You probably don't want to merge this change. It is not required, and can cause an issue on iOS if people click on the customization link twice without going through onboarding (https://github.com/e-mission/e-mission-phone/commit/df672bd7433d87893d1b8c08373e0d63d3a0e0b5). I removed it from the `opentoall` branch, will also remove from `cci-berkeley`

Reply:
Do you mean the `androidDefaultCollectionConfig` part or `iosDefaultCollectionConfig` or both?